R/predictors.R

```diff
@@ -59,7 +59,9 @@
 #' @keywords internal
 getPred_baseflow <- function(data, metadata, method=c("hysep","1p digital filter","2p digital filter"), 
                              da, select, alpha, BFImax, ...) {
-  
+  #silly thing needed to pass R CMD check
+  # from http://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check
+  is.Date=is.POSIXt=hysep=NULL
```

Comment:
This is fixed here https://github.com/miguelcleon/loadflex/blob/master/R/predictors.R#L60